scripts/evergreen/e2e/single_e2e.sh

```diff
@@ -142,6 +147,22 @@ deploy_test_app() {
         helm_params+=("--set" "omDebugHttp=true")
     fi
 
+    if [[ -n "${helm_oci_regisry:-}" ]]; then
```

Comment:
are those values optional or should be expected to be present?

Reply:
these values must be set for evergreen patches but will not be set for local tests. we are trying to run the tests image here that we don't do when we run the tests locally so we should be good in always expecting this to be set.

Reply:
Yes, that's right. I will make the change.